src/components/studyList/StudyList.styl

```diff
@@ -194,6 +205,40 @@ placeholder-color(c)
             border-bottom: 1px solid var(--ui-gray-lighter)
             background-color: var(--table-hover-color)
 
+.form-control
```

Comment:
Form-control should probably be in a more generic `.styl` file

Reply:
Form-control is being used only in `StudyList` component, should we move this anyway?

Reply:
Just to be safe, I would appreciate it if it were part of some styles at a higher level up. It'll make it easier to notice/change as we pull in and style form components as "elements"

Reply:
Ok, I will move it